cf/test/test_Data.py

```diff
@@ -2493,27 +2493,6 @@ def test_Data_section(self):
         self.assertEqual(key, (None, None, None))
         self.assertTrue(value.equals(d))
 
-    @unittest.skipIf(TEST_DASKIFIED_ONLY, "Needs reconstruct_sectioned_data")
-    def test_Data_reconstruct_sectioned_data(self):
-        if self.test_only and inspect.stack()[0][3] not in self.test_only:
-            return
-
-        # TODODASK: Write when Data.reconstruct_sectioned_data is
-        #           daskified
-
-    @unittest.skipIf(TEST_DASKIFIED_ONLY, "no attr. 'partition_configuration'")
-    def test_Data_count(self):
-        if self.test_only and inspect.stack()[0][3] not in self.test_only:
-            return
-
-        d = cf.Data(ma)
-        self.assertEqual(d.count(), 284, d.count())
-        self.assertEqual(d.count_masked(), d.size - 284, d.count_masked())
-
-        d = cf.Data(a)
-        self.assertEqual(d.count(), d.size)
-        self.assertEqual(d.count_masked(), 0)
```

Comment:
Did you mean to remove the `test_Data_count` method as well (asking since it doesn't come under the motivation for the PR)?

Reply:
Oops - no I didn't! I'll reinstate it before I merge ...